application/dashboard/trello_service.py

```diff
@@ -59,6 +61,10 @@ class TrelloService(Service):
     api_token = ""
     client = None
 
+    # Ignore reference ([] with at least one character inside) and optional space; capture all that follows as the name
+    # e.g. "[BLAH 002] New measure name" will drop the "[BLAH 002] " and capture "New measure name"
+    REMOVE_INTERNAL_REFERENCE_REGEX = re.compile(r"\[.+?\]\s*(.*)")
```

Comment:
Add a `\A` to match the start of the string?

Reply:
I'm generally happier with `^` to match the start of the string, but from test results it doesn't seem to need it anyway?  I can't think of an example where adding start of string matcher would give a better result than without.  But if you can then let's add it to the test cases.

But if it's the same either way I'm in favour of leaving extra cruft out of regex to keep it as simple as possible.